src/resources/users/UserForm.tsx

```diff
@@ -42,6 +42,7 @@ export default function UserForm({ isEdit }: FormProps): React.ReactElement {
     role: 'rco-user'
   }
   const { record } = useEditContext()
+  const isPowerUser = record?.role === ''
```

Comment:
@TahaKhanAbdalli - we really need to check they are a power user: `rco-power-user`

![image](https://github.com/DeepBlueCLtd/RCO/assets/1108513/4aff0916-808b-4816-9a88-d3cdf159f7fb)